### DISCUSSION TITLE
Unable to recognize files without extensions

### DISCUSSION THREAD

**Discussion by gi-b716** [decaporg/decap-cms#7355] [Q&A] [answered]
I'm trying to edit some files without extension using Decap CMS, but when I set `extension: “”`, the files are not recognized properly.

Why?

**Comment by martinjagodic** [comment, marked as answer]
@gi-b716 `extention` is not supposed to be empty. If you want to omit the extension for formatting reasons, I suggest you use an extention that is not recognized by formatters.

**Comment by eerison** [comment]
Hi @martinjagodic  do you have any example?

It does not get the file content, the field is empty :/

I am trying like this

```diff
  - name: settings
    label: "Settings"
+  format: other-extension
    files:
      - name: redirects
        label: Redirects
        file: public/_redirects
        fields:
          - { label: "Content", name: "body", widget: "text" }
```

the files that I am tring to edit:
https://developers.cloudflare.com/pages/configuration/redirects/
https://developers.cloudflare.com/pages/configuration/headers/
